Lesson7/Task3/Mercedes.java

```diff
@@ -0,0 +1,27 @@
+package Lesson7.Task3;
+
+public class Mercedes extends Car{
+    private String model;
+    private CarEngine engine;
+    private int yearOfProduction;
+
+    public Mercedes(String model, int yearOfProduction, CarEngine engine) {
+        this.model = model;
+        this.yearOfProduction = yearOfProduction;
+        this.engine = engine;
+    }
```

Comment:
так як ці поля є у батьківського класа Car, то тут їх можна приибрати а в конструкторі викликати батьківський консструктор
```suggestion

    public Mercedes(String model, int yearOfProduction, CarEngine engine) {
        super(model, yearOfProduction, engine);
    }
```

Reply:
Виконано, дякую